lib/spack/spack/bootstrap/environment.py

```diff
@@ -146,6 +147,11 @@ def pytest_root_spec() -> str:
     return _root_spec("py-pytest@6.2.4:")
 
 
+def pyruff_root_spec() -> str:
+    """Return the root spec used to bootstrap flake8"""
+    return _root_spec("py-ruff@0.8.0:")
```

Comment:
Why 0.8?

Reply:
It is the version that uses Python 3.9 by default as the target python version.

Eventually we would probably want to pin the version to avoid breaking linting changes. I didn't here since this is still experimental.

Reply:
Why would we want Python 3.9?

Reply:
Because 3.8 is EOL...

Reply:
Spack still supports Python 3.6 unfortunately...

Reply:
I would actually pick 0.9+, 0.8 and 0.9 have different styles and will reformat each other, they aren't backwards compatible like 0.7 and 0.8 were.